dotCMS/src/main/webapp/html/portlet/ext/cmsmaintenance/view_cms_maintenance.jsp

```diff
@@ -1428,33 +1430,57 @@ dd.leftdl {
                     <th><%= LanguageUtil.get(pageContext,"Import/Export-dotCMS-Content") %></th>
                     <th style="text-align:center;white-space:nowrap;" width="350"><%= LanguageUtil.get(pageContext,"Action") %></th>
                 </tr>
-                <tr>
-                    <td><%= LanguageUtil.get(pageContext,"Backup-to-Zip-file") %></td>
-                    <td style="text-align:center;white-space:nowrap;">
-						<div class="inline-form">
-							<button dojoType="dijit.form.Button" onClick="doCreateZip('true');" iconClass="backupIcon">
-							   <%= LanguageUtil.get(pageContext,"Backup-Data-Only") %>
-							</button>
-							<button dojoType="dijit.form.Button" onClick="doCreateZip('false');" iconClass="backupIcon">
-							  <%= LanguageUtil.get(pageContext,"Backup-Data/Assets") %>
-							</button>
-						</div>
-                    </td>
-                </tr>
+                <% if (Config.getBooleanProperty("ALLOW_STARTER_ZIP_GENERATION_ON_DISK", false)) { %>
+                    <tr>
+                        <td><%= LanguageUtil.get(pageContext,"Backup-to-Zip-file") %></td>
+                        <td style="text-align:center;white-space:nowrap;">
+                            <div class="inline-form">
+                                <button dojoType="dijit.form.Button" onClick="doCreateZip('true');" iconClass="backupIcon">
+                                   <%= LanguageUtil.get(pageContext,"Backup-Data-Only") %>
+                                </button>
+                                <button dojoType="dijit.form.Button" onClick="doCreateZip('false');" iconClass="backupIcon">
+                                  <%= LanguageUtil.get(pageContext,"Backup-Data/Assets") %>
+                                </button>
+                            </div>
+                        </td>
+                    </tr>
+                <% } %>
                 <tr>
                     <td><%= LanguageUtil.get(pageContext,"Download-Zip-file") %></td>
                     <td style="text-align:center;white-space:nowrap;">
 						<div class="inline-form">
-							<button dojoType="dijit.form.Button" onClick="doDownloadZip('true');" iconClass="downloadIcon">
+							<button dojoType="dijit.form.Button" onClick="location.href='/api/v1/maintenance/_donwloadStarter'" iconClass="downloadIcon">
```

Comment:
there is a typo on `_donwloadStarter`. Switch `nw`, otherwise the button link will be broken

Reply:
Fixed